projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/merchandising-carousel.e2e-spec.ts

```diff
@@ -86,6 +85,26 @@ describe.skip('Merchandsing Carousel', () => {
     ).as('strategyProductsApiRequest');
   });
 
+  it('should send two requests when on the homepage for the first time', () => {
+    testHomePage();
+
+    cy.get('@strategyProductsApiRequest').then(request => {
+      expect(request).to.not.have.property('requestHeaders.consent-reference');
+    });
+
+    merchandisingCarousel.acceptConsent();
+
+    merchandisingCarousel.verifyRequestToStrategyService(strategyRequestAlias, {
```

Comment:
fails here because when clicking on the banner to 'accept all', no requests are being made, therefore the `cy.wait` at the beginning of this function fails automatically

Reply:
Main question: is it right to call this function?